__tests__/app.test.js

```diff
@@ -439,3 +439,41 @@ describe("/api/users", () => {
         });
     });
 });
+
+describe("/api/comments/:comment_id", () => {
+    describe("DELETE", () => {
+        test("Returns a status 204 with no content", () => {
+            return request(app).delete('/api/comments/1')
+            .expect(204)
+            .then(( { body }) => {
+                expect(body).toEqual({});
+            });
+        });
+        test("Database no longer contains the comment with the specified ID", () => {
+            return request(app).delete('/api/comments/1')
+            .expect(204)
+            .then(() => {
+                return db.query('SELECT * FROM comments;')
```

Comment:
This is great thinking, but it would be much better if you utilised the GET commentById endpoint you built earlier and expected a 404.

Reply:
Hi Rose, thank you for your help! 😄  I don't think we did a 'GET comment by comment ID' endpoint (unless I've reallllly gone wrong somewhere haha) so I have used 'GET comment by article ID' (GET article/:article_id/comments ) instead. Used this to get the comments for the article_id of the deleted comment, then expected a 200 and checked the returned array length was one less than before. Is that okay?